cubic-server/command-parser/commands/help.cpp

```diff
@@ -0,0 +1,23 @@
+#include "help.hpp"
+
+namespace help {
+
+    void help(std::vector<std::string>& args) {
+        logging::Logger::get_instance()->info("/help [<command>]");
```

Comment:
Why are you not calling the macro ?

Reply:
Ok, so I just saw how the macro works, we need to change that I think. x)

Reply:
We will change that later, not really a big deal right now